src/main/scala/com/elsevier/soda/SodaService.scala

```diff
@@ -183,6 +186,42 @@ class SodaService {
             coveredText.trim().length() > 3 && !stopwords.contains(coveredText)    
         }).toList
     }
+
+    def fuzzySearch(text: String, lexName: String, fuzzinessValue: Int): List[Annotation] = {
+        val words = text.split(" ")
+        // run each of these phrases against
+        val tags = ArrayBuffer[Annotation]()
+        var currentStart = 0
+        words.foreach(word => {
+            val params = new ModifiableSolrParams()
+            params.add(CommonParams.Q, "tagname_str:" + word + "~" + fuzzinessValue)
```

Comment:
"word" ought to be escaped.  Use `org.apache.solr.client.solrj.util.ClientUtils#escapeQueryChars`

Reply:
@dsmiley done.